app/scripts/directives/overview/dc.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('overviewDeploymentConfig', function($filter,
+                                         DeploymentsService,
+                                         MetricsService,
+                                         Navigate) {
+    return {
+      restrict: 'E',
+      // Inherit scope from OverviewController. This directive is only used for the overview.
+      // We want to do all of the grouping of resources once in the overview controller watch callbacks.
+      scope: true,
+      templateUrl: 'views/overview/_dc.html',
+      link: function($scope) {
+        if (!window.OPENSHIFT_CONSTANTS.DISABLE_OVERVIEW_METRICS) {
+          MetricsService.isAvailable(true).then(function(available) {
+            $scope.showMetrics = available;
+          });
+        }
+
+        $scope.viewPodsForDeployment = function(deployment) {
+          if (_.isEmpty($scope.podsByDeployment[deployment.metadata.name])) {
+            return;
+          }
+
+          Navigate.toPodsForDeployment(deployment);
+        };
+
+        $scope.getHPA = function(rcName, dcName) {
```

Comment:
Wonder if we should move this to overview.js so we don't have to repeat it here and in overview/service.js.